### PR TITLE
Pillow macos

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Checklist
* [ ] Added an entry in `doc/news/`. 
* [ ] Added a test for this change.
* [ ] Updated the documentation.

Fixxes support for Macos (see #196 ).

### CODE REVIEW

**Comment by linuxrider** [comment]
fixes #196 
@DunklesArchipel Maybe it would be good to documet this workaround for the users on macos.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -13,23 +13,31 @@ jobs:
...
         run: |
-          mamba env update --quiet -n test -f environment.yml
-          conda list
+          mamba info
+          mamba list
         if: ${{ matrix.source == 'conda' }}
+      - name: workaround for libtiff.5.dylib on macos
+        shell: bash -l {0}
+        run: |
+          ln -s /Users/runner/miniconda3/envs/test/lib/python${{ matrix.python }}/site-packages/PIL/../../../libtiff.6.dylib /Users/runner/miniconda3/envs/test/lib/python${{ matrix.python }}/site-packages/PIL/../../../libtiff.5.dylib
```
Comment (saraedum) [inline .github/workflows/test.yml:39]:
```suggestion
          ln -s /Users/runner/miniconda3/envs/test/lib/libtiff.6.dylib /Users/runner/miniconda3/envs/test/lib/libtiff.5.dylib
```

This feels quite wrong. This should be fixed upstream by fixing the pillow package in conda-forge.

File: doc/news/pillow-macos.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+**Fixed:**
+
+* Fixed support for Macos (see issue #196).
```
Comment (saraedum) [inline doc/news/pillow-macos.rst:3]:
```suggestion
* Fixed support for macOS (see issue #196).
```

**Comment by saraedum** [comment]
What's the problem actually?

**Comment by linuxrider** [comment]
Maybe this is related?
https://github.com/conda-forge/pillow-feedstock/issues/135

**Comment by DunklesArchipel** [comment]
@linuxrider is this resolved with #204 ?

**Comment by linuxrider** [comment]
Yes.

**Comment by DunklesArchipel** [comment]
Resolved in #204

**Pull request closed without merging**